src/components/Routing.js

```diff
@@ -99,17 +100,27 @@ class Routing extends Component {
         // this.addTranslationsForActiveLanguage(defaultLanguage)
     }
 
-    componentDidMount = () => {
+    componentDidMount = async () => {
         const { refreshAccount, handleRefreshUrl, history, clearAlert, clear, handleRedirectUrl, handleClearUrl } = this.props
         
         handleRefreshUrl()
-        refreshAccount()
-
-        history.listen(() => {
+        try {
+            await refreshAccount()
+        } catch (e) {
```

Comment:
why is this change needed?

Reply:
Account's are not created until after recovery method.

The user can sometimes enter into a state where they have a "temp account" and they are in profile or some other route. We need to catch any exceptions to refresh account and push users without a legitimate account to the `/create` path.